types/dictionary.d.ts

```diff
@@ -3,71 +3,24 @@ declare module "fastly:dictionary" {
    * Class for accessing [Fastly Edge Dictionaries](https://docs.fastly.com/en/guides/about-edge-dictionaries).
    *
    * **Note**: Can only be used when processing requests, not during build-time initialization.
-   * 
-   * @example
-   * <script async defer src="https://fiddle.fastly.dev/embed.js"></script>
-   * In this example we have an Edge Dictionary named 'animals' and we return the 'cat'
-   * entry as the response body to the client.
-   * 
-   * <script type="application/json+fiddle">
-   * {
```

Comment:
Same here -- 
This big example comment is not useful in IDE's -- this is a legacy example comment from when our documentation website was generated from the type definitions

Reply:
Looks good. Thank you for taking care of this.